src/DOM/HTML/Indexed/LengthLiteral.purs

```diff
@@ -0,0 +1,12 @@
+module DOM.HTML.Indexed.LengthLiteral where
+
+import Prelude (show, (<>))
+
+data LengthLiteral
+  = Pixels Int
```

Comment:
I actually think this may be unnecessary? I think in `HTML5` these are always pixels (percent is deprecated for CSS).